rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationRector/Fixture/my_form_type.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Symfony\Component\Form;
+
+interface FormTypeInterface
+{
+    /**
+     * @return string|null
+     */
+    public function getParent();
+}
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationRector\Fixture;
+
+final class MyFormType implements FormTypeInterface
```

Comment:
You can register `FormTypeInterface` to config https://github.com/rectorphp/rector-src/blob/main/rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationRector/config/configured_rule.php